patches/gcc/13.2.0/001-safe-ctype.patch

```diff
@@ -0,0 +1,140 @@
+From: Francois-Xavier Coudert <fxcoudert@gcc.gnu.org>
```

Comment:
Why not use gcc 13.3? It seems like this patch is included in 13.3.

Reply:
I wanted to make sure we have the same versions as in ci-builder (plus these fixes for the build)